packages/core/src/Tooltip/Tooltip.stories.tsx

```diff
@@ -45,94 +46,100 @@ export const WithoutAnchors = () => (
 export const WithAnchors = () => (
   <FlexColumn justifyContent='space-between' wrap='wrap'>
     <Box width={300} p={2} my={2}>
-      <Tooltip top left bg='blue' color='white'>
+      <Tooltip top left color='primary'>
         top left tooltip
       </Tooltip>
       <div>some text</div>
     </Box>
     <Box width={'300px'} p={2} mb={5}>
       <div>some text</div>
-      <Tooltip bottom left bg='red' color='white'>
+      <Tooltip bottom left color='error'>
         bottom left tooltip
       </Tooltip>
     </Box>
     <Box width={'300px'} p={2} mb={55}>
-      <InputField
-        icon='circleInfo'
-        color='blue'
-        label='Email Address'
-        defaultValue='albus.dumbledore@priceline.com'
-        id='form-field-3'
-        placeholder='example@test.com'
-      />
-      <Tooltip bottom left bg='blue' color='white'>
+      <FormField>
```

Comment:
InputField did not work like this, it's just a regular FormField so I've updated these stories to provide better/more accurate examples of what you want to do.

Reply:
We don't have an InputField component, it's just a re-export of FormField https://github.com/priceline/design-system/blob/main/packages/core/src/FormField/index.ts#L3